### PR TITLE
Add: Faq page

### LINKED ISSUE #23: Create FAQ section with accordion
please allow me to work on it under jwoc tag!

### PR DESCRIPTION
**Pull request by AaadityaG**
Closes #23 

https://github.com/ritika728/Wellness_Wave/assets/114663382/1404e0f9-ed7c-4dae-ba94-f432f6a748cd

## Please add JWoC tag!

### CODE REVIEW

**Comment by ritika728** [comment]
@AaadityaG please add a heading: 'Frequently Asked Questions' between the two boxes

File: html_files/faq.html
Comment (ritika728) [inline html_files/faq.html:1]: Here, as majority of the code is commented, please remove it for increased code readability.

**Review by AaadityaG — COMMENTED** [review]
done with above commit

**Comment by ritika728** [comment]
@AaadityaG Thank you for your contribution! 🚀

**Pull request merged**